Jenkinsfile

```diff
@@ -10,13 +10,16 @@ pipeline {
             }
           }
           steps {
+            echo "Building environment..."
             sh """
                virtualenv .
                source bin/activate
                pip install 'flake8==3.3.0' 'coverage==4.3.4' 'pytest==3.0.6' 'pytest-cov==2.4.0' 'futures==3.0.5' 'requests==2.13.0' 'six' 'wheel' 'pyyaml>=3.10,<=3.12' 'pyOpenSSL' 'importlib' 'Jinja2==2.9.6'
-               pytest
-               flake8 
-            """
+               """
+            echo "Testing with Pytest..."
+            sh 'source bin/activate && pytest'
```

Comment:
I wonder if you could just do `bin/pytest` rather than `source bin/activate && pytest`

Reply:
that should work. I'll update the PR